Assets/Tests/EditMode/AuthTests/AuthControllerTests.cs

```diff
@@ -15,16 +15,17 @@ public class AuthControllerTests
         [TestCaseSource(nameof(GetAuthTypes))]
         public void OnAuthTypeChosen_ShouldCacheAccessTokenProvider_ObtainedFromStrategy(AuthType authType)
         {
-            bool bCached = false;
+            var bCached = false;
             
             var usedTokenProvider = new Mock<IAccessTokenProvider>().Object;
             var cacherMock = new Mock<IObjectCacher>();
             cacherMock.Setup(a => a.CacheObject(usedTokenProvider))
                 .Callback<IAccessTokenProvider>((_) => bCached = true);
             var providersStrategyMock = new Mock<IAccessTokenProvidersStrategy>();
             providersStrategyMock.Setup(s => s.GetAccessTokenProviderAsync(authType)).ReturnsAsync(usedTokenProvider);
+
             var authViewMock = new Mock<IAuthView>();
-            authViewMock.Setup(a => a.Login(authType)).Raises(a => a.AuthTypeChosen += null, this, authType);
+            authViewMock.Setup(a => a.Login(authType)).Raises(a => a.AuthTypeChosen += null, authType);
```

Comment:
nit: By the way, it is easier to use Raise method instead of Raises, as it allows you to raise some event on mock without using any methods of mocked interface.